ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Docker/tasks/sles.yml

```diff
@@ -22,7 +24,18 @@
     - docker-ce-cli
     - containerd.io
   when:
+    - ansible_distribution_major_version == "15"
     - ansible_architecture == "x86_64" or ansible_architecture == "s390x"
   tags:
     # TODO: Package installs should not use latest
     - skip_ansible_lint
+
+- name: Install docker in SLES 12
+  package:
+    update_cache: yes
+    name: ['containerd', 'runc', 'docker']
+    state: latest
+  when:
+    - ansible_distribution == "SLES"
+    - ansible_distribution_major_version == "12"
+    - ansible_architecture != "aarch64"  #no docker support for sles 12 aarch64
```

Comment:
Is this exclusion really only for aarch64? Will it install on s390x/ppc64le? If not we should switch this to be "ansible_architecture=x64"

Reply:
Yep. Only for aarch64. There is docker support in s390x and ppc64le.
ppc64le: https://scc.suse.com/packages/25502809
s390x: https://scc.suse.com/packages/25502803

Tested on x86, ppc64le and s390x.